cmd/krel/cmd/release_notes.go

```diff
@@ -787,6 +787,17 @@ func releaseNotesJSON(tag string) (string, error) {
 		return "", errors.Wrap(err, "parsing semver from tag string")
 	}
 
+	logrus.Info("Cloning kubernetes/sig-release to read mapping files")
+	sigReleaseRepo, err := git.CleanCloneGitHubRepo(
+		git.DefaultGithubOrg, git.DefaultGithubReleaseRepo, true,
+	)
+	if err != nil {
+		return "", errors.Wrap(err, "performing clone of k/sig-release")
+	}
+	defer func() {
+		err = sigReleaseRepo.Cleanup()
```

Comment:
nitpick: I don't think assigning `err` here will have any effect unless you have a named return value for the encompassing function

Reply:
Good catch @hasheddan ! I found two like that in the file. They are fixed now.
Thanks !